examples/dataset/crossing_pedestrian_detection/crossing_pedestrian_detection/upload_dataset.py

```diff
@@ -28,6 +31,8 @@
 
 from kolena.dataset import upload_dataset
 
+TRANSPORT_PARAMS = {"client": boto3.client("s3", config=Config(signature_version=botocore.UNSIGNED))}
```

Comment:
Any reason not to put this into constants, since this is used in both upload_dataset and upload_results?

Reply:
Updated with https://github.com/kolenaIO/kolena/pull/625/commits/735b2961f49cfb0549b9f3a4d7b5cb5105488203 to refactor into constant import. Also shamelessly plugged in https://github.com/kolenaIO/kolena/pull/625/commits/f6a520d1d15fbf36562e7b9d1483c3e60c57ae03 from https://github.com/kolenaIO/kolena/pull/678